content/en/docs/reference/access-authn-authz/rbac.md

```diff
@@ -209,14 +218,18 @@ rules:
   verbs: ["update", "get"]
 ```
 
-Note that `create` requests cannot be restricted by resourceName, as the object name is not known at
-authorization time. The other exception is `deletecollection`.
+{{< note >}}
+You cannot restrict `create` or `deletecollection` requests by resourceName. For `create`, this
+is limitation is because the object name is not known at authorization time.
+{{< /note >}}
+
 
 ### Aggregated ClusterRoles
 
-As of 1.9, ClusterRoles can be created by combining other ClusterRoles using an `aggregationRule`. The
-permissions of aggregated ClusterRoles are controller-managed, and filled in by unioning the rules of any
-ClusterRole that matches the provided label selector. An example aggregated ClusterRole:
+You can _aggregate_ several ClusterRoles into one combined ClusterRole, using an `aggregationRule`.
+The control plane automatically grants access by taking union of all rules that match a label
```

Comment:
I'm not sure what "The control plane automatically grants access" refers to... aggregation doesn't involve automatic granting of permissions

Reply:
I was thinking of https://github.com/kubernetes/kubernetes/blob/master/pkg/controller/clusterroleaggregation/clusterroleaggregation_controller.go